flamingpy/cv/ops.py

```diff
@@ -193,7 +232,7 @@ def apply_noise(self, model={}, rng=default_rng()):
             "sampling_order": "initial",
             "perfect_inds": perfect_inds,
         }
-        model = {**default_model, **model}
+        model = {**default_model, **model} if model else default_model
```

Comment:
In line of line 225, is this necessary? I suggest
```suggestion
        model = {**default_model, **model}
```